cssom-view-1/Overview.bs

```diff
@@ -1136,20 +1134,17 @@ The <dfn attribute for=CaretPosition>offset</dfn> attribute must return the <a>c
 The <dfn method for=CaretPosition>getClientRect()</dfn> method must follow these steps,
 aborting on the first step that returns a value:
 
-1. If <a>caret range</a> is not null:
-    1. Let <var>list</var> be the result of invoking the {{Range/getClientRects()}} method on the range.
-    1. If <var>list</var> is empty, return null.
-    1. Return the {{DOMRect}} object in <var>list</var> at index 0.
 1. If <a>caret node</a> is a text entry widget that is a replaced element,
     and that is in the document,
     return a [=scaled=] {{DOMRect}} object for the caret in the widget
     as represented by the <a>caret offset</a> value.
     The <a>transforms</a> that apply to the element and its ancestors are applied.
-1. Return null.
+1. Otherwise:
 
-Note: This {{DOMRect}} object is not <a spec=html>live</a>.
+    1. Let <var>caretRange</var> be a collapsed {{Range}} object whose [=range/start node=] and [=range/end node=] are set to <a>caret node</a>, and whose [=range/start offset=] and [=range/end offset=] are set to <a>caret offset</a>.
```

Comment:
This seems okay for now given this is how both Firefox and Chrome/Edge implement this. However, it seems less than ideal to have to create a range in order to get a bounding client rect. That range will still need to be maintained across mutations until it is garbage collected, which may not happen immediately. Is there a different option we can pursue that doesn't use a live range at all?

Reply:
Thanks. I am only aware of such way to get DOMRect from (node, offset). Totally agree that we should update the spec if we have better way to do that.